Procfile

```diff
@@ -1 +1 @@
-web: coffee app.coffee
+web: npm run start:prod
```

Comment:
`npm start` should work... if it doesn't `rm -rf node_modules/` and `npm install`